packagemanifests/0.20.0/deviceplugin.intel.com_dsadeviceplugins.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Do we really want to keep the generated packagemanifests in the repo?

I thought they were of no use in the repo and didn't include them for the 0.19 release.

Reply:
They're quite useful for me to see the difference between previous and new versions, but this is minor. I can remove them if needed.

Reply:
@mythi wdyt?

Reply:
The difference can be checked against the manifests published at the operatorhub.

Reply:
True, but you have to copy manifests into the certain location of the community-operators clone to do that.

Reply:
> I thought they were of no use in the repo and didn't include them for the 0.19 release.

@rojkov has a good point here. If it's just a copy of the CRD and clusterserviceversion that are in this repo already, I'd drop `packagemanifest/*`.

Reply:
ok, dropped